airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -46,19 +46,9 @@ def session_scope(session):
 
     class DelayAlertMetaData(Base):
         __tablename__ = "delay_alert_metadata"
-        dag_id = Column(String, primary_key=True)  # hack to have dag_id as PK
-        task_id = Column(String)
-        cadence = Column(String)
-        severity = Column(String)
-        alert_target = Column(String)
-        alert_external_classification = Column(String)
-        alert_report_classification = Column(String)
-        sla_interval = Column(String)
-        sla_time = Column(String)
-        group_pagerduty = Column(String)
-        group_business_line = Column(String)
-        inhibit_rule = Column(String)
-        latest_successful_run = Column(UtcDateTime)
+        __table_args__ = {"autoload": True}
+        dag_id = Column(String, primary_key=True)
+        latest_successful_run = Column(UTCDateTime)
```

Comment:
Keep only the ones that need to be declared explicitly. Leave the rest autoloaded.